lib/matplotlib/legend.py

```diff
@@ -110,9 +110,33 @@ def _update_bbox_to_anchor(self, loc_in_canvas):
 
 
 _legend_kw_doc = '''
-loc : int or string or pair of floats, default: :rc:`legend.loc` ('best' for \
-axes, 'upper right' for figures)
-    The location of the legend. Possible codes are:
+loc : str or pair of floats, default: :rc:`legend.loc` ('best' for axes, \
+'upper right' for figures)
+    The location of the legend.
+
+    The strings
+    ``'upper left', 'upper right', 'lower left', 'lower right'``
```

Comment:
I know that strings don't usually get double-backquoted in the docs, but I did so on purpose here as it seems better to add a bit of emphasis to the list of valid values.  (The linebreaks are for the same reason.)